WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -312,7 +312,10 @@ private void autofillFromBuildConfig() {
     }
 
     public boolean canAutofillUsernameAndPassword() {
-        return EditTextUtils.getText(mUsernameEditText).isEmpty() && EditTextUtils.getText(mPasswordEditText).isEmpty();
+        return EditTextUtils.getText(mUsernameEditText).isEmpty()
```

Comment:
These EditTexts should only be null if `onCreateView()` hasn't been called, which means the fragment isn't attached. Perhaps an `isAdded()` check would make sense in `onCredentialsRetrieved()`, then?